.github/workflows/tab-finder.yml

```diff
@@ -0,0 +1,39 @@
+# Find tabs in modified text files
+name: Tab finder
+on: [push, pull_request]
+env:
+  MAIN_BRANCH: dev
+jobs:
+  build:
+    name: Tab finder
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # needed to get the full history and pull from upstream
+      - name: Pull from upstream
+        run: |
+          # Update from upstream to avoid including missing commits in the diff
+          git config --global user.name "Nemo" # required on some servers
+          git config pull.rebase false
+          git remote add upstream https://github.com/AliceO2Group/AliceO2.git || exit 1
+          git pull upstream ${{ env.MAIN_BRANCH }} || exit 1
+      - name: Find tabs
+        run: |
+          # Find tabs in modified text files and show where they are
+          status=0
+          # loop over changed files
+          for f in $(git diff --name-only upstream/${{ env.MAIN_BRANCH }}); do
```

Comment:
You should probably use `git diff --diff-filter d` here, to avoid checking deleted files.

I'd also use `"$(git merge-base HEAD ${{ github.event.pull_request.base.sha }})"` instead of `upstream/${{ env.MAIN_BRANCH }}`, so you don't end up checking unrelated changes in dev, which weren't introduced by the PR in question. (See also: [code-formatting.yml](https://github.com/AliceO2Group/AliceO2/blob/d08978eabc0702203c6a6141bdb9650f11cd1a8b/.github/workflows/code-formatting.yml#L44-L48))

Reply:
`github.event.pull_request` is available only in pull requests. This way the action runs for a push as well.

Reply:
Ah, OK. How about using the merge-base of dev and the HEAD of the PR or push? I still think that'd be the correct thing to do for PRs, but does that cause problems for plain pushes?

Reply:
Done